docs/docs/reference/02-manual-installation-launch.md

```diff
@@ -0,0 +1,289 @@
+runtime application image. In this example, a manifest file saved as `aica-application.toml` is used to build an image
+with the name `aica-runtime`.
+
+```shell
+docker build -f aica-application.toml -t aica-runtime .
+```
+
+The command `docker image ls | grep aica-runtime` should then list the `aica-runtime` image.
+
+
+## Starting the application container
+
+You can start the AICA application container with the following command.
+
+:::note
+
+Change `/path/to/license` in the command below to the location of the personal AICA developer license assigned to you.
+For example, use `~/.aica-license.toml` to keep the license file hidden in the home folder.
+
+:::
+
+
+<Tabs groupId="os">
+<TabItem value="linux" label="Linux">
+
+```bash
+docker run -it --rm \
+  --privileged \
+  --net=host \
+  -v /path/to/license:/license:ro \
+  aica-runtime
+```
+
+</TabItem>
+<TabItem value="mac" label="macOS">
+
+```bash
+docker run -it --rm \
+  --privileged \
+  -p 8080:8080 -p 5000:5000 \
+  -v /path/to/license:/license:ro \
+  aica-runtime
+```
+
+</TabItem>
+</Tabs>
+
+When the container starts up, it will generate some initial output in the terminal window that should look something
+like the example below:
+
+```console
+[2023-12-11 14:48:38 +0000] [65] [INFO] Starting gunicorn 21.2.0
+[2023-12-11 14:48:38 +0000] [65] [INFO] Listening at: http://0.0.0.0:5000 (65)
+[2023-12-11 14:48:38 +0000] [65] [INFO] Using worker: gthread
+[2023-12-11 14:48:38 +0000] [69] [INFO] Booting worker with pid: 69
+[INFO] [1702306118.970110638] [StateEngine.ServiceHandler]: Initializing state engine services
+[INFO] [1702306118.977229190] [state_engine]: No initial application provided. Use the state engine service interface to set, initialize and start an application.
+[INFO] [1702306119.566053151] [StateEngineInterface]: Successfully connected to Dynamic State Engine services
+```
+
+:::info
+
+If there are any errors, check that the license file is valid and has been mounted correctly. For example, the following
+error would be shown from a correctly mounted but invalid license file:
+
+```console
+[ERROR] [1702306228.377201011] [licensing]: Error: license is invalid (ERR - license is invalid), please check that it is correct
+```
+
+Contact AICA support if the container does not start correctly despite a valid license file.
+
+:::
+
+## Stopping the application container
+
+To shut down the AICA application container at any time, press CTRL+C in the original terminal window. Alternatively,
+to stop the application container from a different terminal window, look up the container name
+with `docker container ps` and then run `docker container stop <container_name>`.
+
+## Access the Developer Interface
+
+Visit [localhost:8080](http://localhost:8080) in the browser while the container is running to view the Developer
+Interface.
+
+## Access the REST API
+
+Visit [localhost:5000](http://localhost:5000) to see the Swagger UI and documentation for the REST API.
```

Comment:
```suggestion
Visit [localhost:8080/api](http://localhost:8080/api) to see the Swagger UI and documentation for the REST API.
```
That should be the canonical URL. It's technically broken now but we will fix it next version.